### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**
Screenshot 2022-11-08 at 14 13 53
Screenshot 2022-11-08 at 14 15 15
Screenshot 2022-11-08 at 14 15 08
Screenshot 2022-11-08 at 14 15 01
Screenshot 2022-11-08 at 14 14 04

### CODE REVIEW

File: taxi/forms.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        raise ValidationError("First 3 characters must be uppercase letters.")
+    elif not license_number[3:].isdigit():
+        raise ValidationError("Last 5 characters must be digits.")
+    return license_number
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
+        widget=forms.CheckboxSelectMultiple,
```
Comment (taras-andruschenko) [inline taxi/forms.py:22]: if you look at your screens you will see that checkbox widget hasn't been added.. could you check it, please?

**Review by taras-andruschenko — CHANGES_REQUESTED** [review]
Look's good, but test your checkbox widget, please

File: taxi/forms.py (hunk 1 above)
Comment (kateryna3k) [reply taxi/forms.py:22]: Screenshot 2022-11-09 at 13 36 17
fixed) forgot to add `form_class` to the views

**Comment by taras-andruschenko** [comment]
Well done)

**Review by taras-andruschenko — APPROVED** [review]

**Review by MasterpieceElbow — APPROVED** [review]
GJ!

**Pull request closed without merging**